cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminDesignTime.feature

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Sysadmin
+Feature: Sysadmin - Validate system admin page design time scenarios
+
+  @Sysadmin
+  Scenario:Validate user is able to create new system preferences and able to delete the added system preferences successfully
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the "System Admin" link from menu
+    Then Click on the Configuration link on the System admin page
+    Then Select "systemPreferences" option from Configuration page
+    Then Click on edit system preferences to set system preferences
+    Then Enter key value pairs to set preferences: "keyValue" with values from json: "systemPreferences1"
+    Then Click on Save and Close button to save the preferences
+    Then Select "systemPreferences" option from Configuration page
+    Then Click on edit system preferences to set system preferences
+    Then Click on Delete button to delete the added preferences
+    Then Click on Save and Close button to save the preferences
+
+  Scenario:Validate user is able to add multiple system preferences inside system admin successfully
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the "System Admin" link from menu
+    Then Click on the Configuration link on the System admin page
+    Then Select "systemPreferences" option from Configuration page
+    Then Click on edit system preferences to set system preferences
+    Then Enter key value pairs to set preferences: "keyValue" with values from json: "systemPreferences2"
+    Then Click on Save and Close button to save the preferences
+    Then Click on edit system preferences to set system preferences
+    Then Click on Delete button to delete the added preferences
+    Then Click on Delete button to delete the added preferences
+    Then Click on Save and Close button to save the preferences
+
+  Scenario:Validate user is able to successfully reload system artifacts using reload
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the "System Admin" link from menu
+    Then Click on the Configuration link on the System admin page
+    Then Click on Reload System Artifacts from the System admin page
+    Then Click on Reload button on popup to reload the System Artifacts successfully
+    Then Select "systemPreferences" option from Configuration page
+
+  Scenario:Validate user is able to open compute profile page and select a provisioner
```

Comment:
Again we have selected the provisioner from UI but how do we validate it ?

Reply:
Here we are adding only the design time scenarios we will be adding the validation steps when the requirement for creation with end to end tests is planned in upcoming tasks. Now we are only checking the basic sanity checks. It's an on going task for addition at framework level.

Reply:
Added the validation step.

Reply:
Rest of the changes looks good, I am not actually sure about this one. If we will be creating more scenarios for this use case then its okay but for creating a compute profile this is the only scenario we may have to complete it.

1. Selected an provisioner ```existingdataproc``` - Done
2. Configure the provisioner after filling the details create it.
3. Then check if the provisoiner which we created is present under computer profile list with same name.

This will be a complete end to end test case for this I believe.

Reply:
For now added the validation as suggested to check provisioner created.